keyboards/fortitude60/rev1/config.h

```diff
@@ -0,0 +1,94 @@
+/*
+Copyright 2017 Danny Nguyen <danny@keeb.io>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#ifndef REV1_CONFIG_H
+#define REV1_CONFIG_H
+
+#include QMK_KEYBOARD_CONFIG_H
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xCB10
+#define PRODUCT_ID      0x1156
+#define DEVICE_VER      0x0100
+#define MANUFACTURER    Pekaso
+#define PRODUCT         The Fortitude60 Keyboard
+#define DESCRIPTION     Split 60 keyboard.
+
+/* key matrix size */
+// Rows are doubled-up
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 6
+
+// wiring of each half
+#define MATRIX_ROW_PINS { D1, D0, D3, B7, B6 }
+#define MATRIX_COL_PINS { F7, F6, F5, B3, B2, B1 }
+
+//#define MATRIX_ROW_PINS { D4, D7, E6, B4, B5 }
+//#define MATRIX_COL_PINS { F6, F7, B1, B3, B2, B6 }
+
+/* COL2ROW or ROW2COL */
+#define DIODE_DIRECTION COL2ROW
+
+/* define if matrix has ghost */
+//#define MATRIX_HAS_GHOST
+
+/* number of backlight levels */
+#ifdef BACKLIGHT_ENABLE
```

Comment:
Keep in mind, by hidding these behind a ifdef that they may not work for the configurator (depending on the default rules.mk for the board, not the keymap).  

Same for RGB.